django_opentracing/tracer.py

```diff
@@ -1,12 +1,26 @@
 from django.conf import settings
 import opentracing
 
+django_tracer = None
+
+def get_tracer():
+    return opentracing.tracer
+
+def get_current_span(request):
+    # this lets django rest framework work seamlessly since they wrap the request
+    if hasattr(request, '_request'):
+        request = _request
+    return django_tracer.get_span(request)
+
+
 class DjangoTracer(object):
     '''
     @param tracer the OpenTracing tracer to be used
     to trace requests using this DjangoTracer
     '''
     def __init__(self, tracer):
+        global django_tracer
```

Comment:
Personally I don't like the idea of having a global here. We have it for other connectors, but for frameworks we have this toplevel-tracer approach, which is most likely easier/better. As before, if there's any need for this change, we can take it into consideration.